client/packages/inventory/src/Stocktake/DetailView/DetailView.tsx

```diff
@@ -1,133 +1,102 @@
 import React, { useCallback, useEffect } from 'react';
 import {
-  TableProvider,
-  createTableStore,
   useEditModal,
   DetailViewSkeleton,
   AlertModal,
   RouteBuilder,
   useNavigate,
   useTranslation,
   DetailTabs,
-  useRowHighlight,
   useBreadcrumbs,
   useSimplifiedTabletUI,
   Box,
+  useNonPaginatedMaterialTable,
+  Groupable,
+  NothingHere,
+  MaterialTable,
+  useIsGroupedState,
 } from '@openmsupply-client/common';
 import { ActivityLogList } from '@openmsupply-client/system';
 import { Toolbar } from './Toolbar';
 import { Footer } from './Footer';
 import { AppBarButtons } from './AppBarButtons';
 import { SidePanel } from './SidePanel';
-import { StocktakeSummaryItem } from '../../types';
 import { StocktakeLineEdit } from './modal/StocktakeLineEdit';
-import { ContentArea } from './ContentArea';
 import { AppRoute } from '@openmsupply-client/config';
-import {
-  StocktakeFragment,
-  StocktakeLineFragment,
-  useStocktakeOld,
-} from '../api';
+import { StocktakeLineFragment, useStocktakeOld } from '../api';
 import { StocktakeLineErrorProvider } from '../context';
-import { isStocktakeDisabled } from '../../utils';
-
-const StocktakeTabs = ({
```

Comment:
this was broken into 3 components, found it easier just to have one now that everything is a bit simpler, and so the file reads a bit more in the order of the component tree